packages/ia-components/sandbox/audio-players/audio-player-main.jsx

```diff
@@ -95,16 +89,25 @@ export default class TheatreAudioPlayer extends Component {
         <ThirdPartyEmbeddedPlayer
           sourceURL={sourceURL}
           title={name}
-
         />
       );
       // updateURL
       if (urlSetterFN) {
         urlSetterFN(trackNumber);
       }
     }
+    const archiveStyle = isExternal ? { visibility: 'hidden' } : { visibility: 'visible' };
 
-    return mediaElement;
+    return (
+      <Fragment>
```

Comment:
**Question**: what's the benefit to using the longhand `<React.Fragment>` rather than the shorthand `<>`? I know that when you want to attach a key to the fragment element, you have to be explicit, but here we're not. Wondering if there's some other reason to use it over the more terse version.

Reply:
I'll re-try the short hand of it, just got used to long form.